build_tools/circle/build_doc.sh

```diff
@@ -2,18 +2,74 @@
 set -x
 set -e
 
-# Introspect the commit to know whether or not we should skip building the
+# Inspect the commit to know whether or not we should skip building the
 # documentation: a pull request that does not change any file in doc/ or
 # examples/ folder should be skipped unless the "[doc: build]" is found the
```

Comment:
Also I believe the information in the check_build_doc.py docstring is a bit more complete, that would be a good idea to move it here.